src/fsharp/TypedTreeBasics.fsi

```diff
@@ -207,6 +207,8 @@ val canAccessFromEverywhere: Accessibility -> bool
 
 val canAccessFromSomewhere: Accessibility -> bool
 
+val canAccessFromSomewhereOutside: 'T -> Accessibility -> bool
```

Comment:
@dsyme what was the train of thought with this first generic parameter?
I'm currently only able to deduce that the top-level `Entity` has the attribute inside the `.Attribs` property.